auth/src/main/java/com/firebase/ui/auth/IdpResponse.java

```diff
@@ -99,6 +110,13 @@ public String getEmail() {
         return mEmail;
     }
 
+    /**
+     * @return true if the user was created in the sign in flow, false if it already existed.
+     */
+    public boolean isNewEmailUser() {
```

Comment:
I think we should delay this feature because with a little more effort (which would be out of scope here) we could do a more complete job with the "new or existing user" problem.

At the time when we acquire the users email (either directly or through an IDP response) we could do a query for providers from email and use that to determine if it's a new or existing user.  It comes at the cost of an extra request but at least it would be correct.

The other option is just to recommend that developers keep an index of known users in the Realtime Database, which is what ~90% of devs are doing  anyway (for various reasons).  We could even provide a FBUI helper function to manage this.

In any case, we should discuss it thoroughly.